components/elm/src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -869,7 +869,8 @@ subroutine CanopyFluxes(bounds,  num_nolakeurbanp, filter_nolakeurbanp, &
             p = filterp(f)
             c = veg_pp%column(p)
             if(.not.veg_pp%is_fates(p)) then
-               if (veg_pp%itype(p) == nsoybean .or. veg_pp%itype(p) == nsoybeanirrig) then
+               ! soybean (crop with N fixation)
+               if (crop(veg_pp%itype(p)) >= 1 .and. nfixer(veg_pp%itype(p)) == 1) then
```

Comment:
crop is still defined as a binary in pftvarcon:
https://github.com/E3SM-Project/E3SM/blob/6ced7eb263abd7be11a1edba1eb0049e5e0cfb6b/components/elm/src/main/pftvarcon.F90#L106

Can you update the definition since this can be larger than 1?

Reply:
No plan for this to be greater than 1. This can be changed following the example of iscft, but would prefer to get it in a later PR.